controllers/cards.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-dynamic-require */
+const path = require('path');
+
+const Card = require(path.join(__dirname, '../models/card'));
+
+module.exports.createCard = (req, res) => {
+  const { name, link } = req.body;
+  Card.create({ name, link, owner: req.user._id })
+    .then((card) => res.send({ data: card }))
+    .catch(() => res.status(500).send({ message: 'Произошла ошибка' }));
+};
+
+module.exports.findCards = (req, res) => {
+  Card.find({})
+    .populate('user')
+    .then((cards) => res.send({ data: cards }));
+};
+
+module.exports.deleteCard = (req, res) => {
+  Card.findByIdAndRemove(req.params.id)
+    .then((card) => {
+      if (card) {
+        res.send({ message: `Карточка c id ${card._id} удалена` });
```

Comment:
Можно лучше: не указывать `id` в сообщении в теле ответа.
Клиент сам говорит какую карточку нужно удалить. Передавать `id` в сообщении актуально только если сервер сам решает какую карточку удалить. В этом случае клиенту важно знать.
Также это позволит сделать сообщения короче, что на клиенте может уменьшить размер нотификаций.
Т.е. достаточно:
```suggestion
        res.send({ message: `Карточка удалена` });
```